axelrod/tests/unit/test_deterministic_cache.py

```diff
@@ -27,6 +28,9 @@ def tearDownClass(cls):
         os.remove(cls.test_save_file)
         os.remove(cls.test_load_file)
 
+    def setUp(self):
+        self.cache = DeterministicCache()
```

Comment:
Does this create a shared cache across all tests? That could cause some confusion.

Reply:
That's a good point. But since `setUp` is called before every test method, `self.cache` is assigned a new, empty cache for each one. Was that your concern?

Reply:
Yep, thanks for the explanation.